python-sdk/nuscenes/eval/predict/metrics.py

```diff
@@ -0,0 +1,436 @@
+# nuScenes dev-kit.
+# Code written by Freddy Boulton, Eric Wolff 2020.
+""" Implementation of metrics used in the nuScenes prediction challenge. """
+import abc
+from typing import List, Dict, Any, Tuple
+
+import numpy as np
+from scipy import interpolate
+
+from nuscenes.predict import PredictHelper
+from nuscenes.predict.input_representation.static_layers import load_all_maps
+from nuscenes.map_expansion.map_api import NuScenesMap
+from nuscenes.eval.predict.data_classes import Prediction
+
+
+def returns_2d_array(function):
+    """Makes sure that the metric returns an array of
+    shape [batch_size, num_modes]"""
```

Comment:
For one liners: Add a space before and after the sentence, ending in `.`.
For multi-liners: Write the text in a separate line.

Reply:
Done. Thanks for suggestion.